pages/api/register.js

```diff
@@ -25,46 +25,10 @@ export default async function handler (request, response) {
         }
       })
       if (newUser) {
-        response.status(200).json({ message: 'New user created.' })
+        response.send(200).json({ message: 'New user created.' })
       } else {
```

Comment:
I think we actually don't need this `else` here. If it cannot create a user because of an error, the error will be thrown earlier anyways, and the `else` wouldn't be reached